src/components/run-job/authoring-panel.module.css

```
@@ -0,0 +1,829 @@
+  display: flex;
+  font-size: 13px;
+  font-weight: 600;
+  gap: 7px;
+  padding: 7px 15px;
+  transition: all 0.3s ease;
+  white-space: nowrap;
+}
+
+.pillMark {
+  align-items: center;
+  border: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)